examples/daal4py/run_examples.py

```diff
@@ -52,9 +54,9 @@
 assert 8 * struct.calcsize('P') in [32, 64]
 
 if 8 * struct.calcsize('P') == 32:
-    logdir = jp(exdir, '_results', 'ia32')
+    logdir = jp(exdirs[0], '_results', 'ia32')
```

Comment:
I am leaving logs in the `examples/daal4py` dir.
Do we need to move it into `examples`?

Reply:
```suggestion
    logdir = jp(jp(rootdir, 'daal4py'), '_results', 'ia32')
```

Reply:
Might be it worth to completely move run_examples script to tests directory? It's a bit strange for running sklernex examples from daal4py directory.

Reply:
I was thinking about it, so that is why I ask. Moving it may require some additional changes. But it is really better, let me try.